schemas/mw-article-edit.js

```diff
@@ -0,0 +1,68 @@
+{
+    "title": "MediaWiki Article Edit",
+    "description": "Represents a MW Article Edit event",
+    "type": "object",
+    "properties": {
```

Comment:
I'm quite ambivalent towards the user info. In the case of anon edits, it will contain the user's IP and we wouldn't want to expose that. On the other hand, having user info in these events can come in quite handy.